src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
The reason I had this in the simulator specific API is that the horrible jitter is a function of noise level * update rate. If you connect a simulator running at 400 Hz the flight-measured noise levels would be perfectly fine, as each peak would be smoothed out before making it back to QGC. Because of only 50 Hz update rate we kind of have to "pre-filter" the noise level to match the bandwidth we have. Consequently the noise scaling is better being done on the per-simulator interface, factoring in the specific limitations.

The only point to not do this is would be if we think that we will never get a simulator capable of doing 200-400 Hz interfaced through QGC. Right now all faster simulators talk directly to the autopilot and bypass QGC.